clients/go/admin/config.go

```diff
@@ -32,6 +32,8 @@ const (
 	AuthTypeClientSecret AuthType = iota
 	// Chooses Proof Key Code Exchange OAuth2 extension protocol (ref: https://tools.ietf.org/html/rfc7636)
 	AuthTypePkce
+	// Chooses an external authentication process
+	AuthTypeExternalProcess
```

Comment:
nit: rename to match the name of the field to fill in? just for convenience...

```suggestion
	AuthTypeExternalCommand
```